packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -154,5 +154,16 @@
     "localVideoCameraSwitcherLabel": "Switch camera",
     "localVideoMovementLabel": "Movable Local Video Tile",
     "localVideoSelectedDescription": "{cameraName} selected"
+  },
+  "fileUploadCards": {
+    "removeFile": "Remove file from uploads"
+  },
+  "fileCard": {
+    "uploadProgress": "upload progress",
```

Comment:
These string attribute names aren't intuitive to me. Just by looking at the names, I can't imagine what they represent.
This also doesn't take into account how phrases differ for each language. Could you consult with other team members who have worked on locale strings to come up with more generic strings and attribute names for said strings?

Reply:
removed this string, was used to indicate the file progress with file name but as the file name is already announced , we don't need this string now